hazelcast/src/hazelcast/util/impl/Executor.cpp

```diff
@@ -84,18 +86,61 @@ namespace hazelcast {
                     return;
                 }
 
+                size_t numberOfWorkers = workers.size();
+                size_t numberOfDelayedRunners = delayedRunners.size();
+
+                logger.finest() << "ExecutorService " << threadNamePrefix << " has " << numberOfWorkers << " workers and "
```

Comment:
Don't we have `isFinest` checks like we have in java?

Reply:
yes, we have it. Since this was during shutdown, I did not think optimization is important at this line. Should we use it?

Reply:
If we add the check as a general rule that would be great. That way we don't need to think if it is necessary or not.